src/themes/zero/shell.css

```
@@ -0,0 +1,1920 @@
+	border: none;
+	background: transparent;
+	color: var(--icon-color);
+	border-radius: 0.5rem;
+	cursor: pointer;
+}
+
+[data-ui-theme='zero'] .z-icon-btn:hover {
+	background: color-mix(in srgb, var(--z-fg) 6%, transparent);
+}
+
+[data-ui-theme='zero'] .z-icon-btn.spin :global(.z-icon) {
```

Stylelint (17.14.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)